iOS_SDK/OneSignalSDK/OneSignalUserTests/SwitchUserTests.swift

```diff
@@ -0,0 +1,463 @@
+import XCTest
+import OneSignalCore
+import OneSignalCoreMocks
+import OneSignalUserMocks
+@testable import OneSignalUser
+
+final class SwitchUserTests: XCTestCase {
```

Comment:
Nit: Might be good to state these are more integration style tests.

Reply:
Renamed class to `SwitchUserIntegrationTests`